src/modules/flow-metatype/js/js_code_start.h

```diff
@@ -720,6 +721,9 @@
     "}\n" \
     "static int\n" \
     "js_metatype_simple_port_process(struct sol_flow_node *node, void *data, uint16_t port, uint16_t conn_id,\n" \
+    "    const struct sol_flow_packet *packet) SOL_ATTR_UNUSED;\n" \
+    "static int\n" \
+    "js_metatype_simple_port_process(struct sol_flow_node *node, void *data, uint16_t port, uint16_t conn_id,\n" \
```

Comment:
I need to set SOL_ATTR_UNUSED at the function prototype. Otherwise gcc complains